docs/modules/ROOT/pages/ios_troubleshooting.adoc

```diff
@@ -34,38 +33,40 @@ Once these have been enabled:
 * Go back to the Logging settings and click btn:[Share log file].
 
 .All iOS App logging settings enabled and set
-image:ios-app-settings-logging.png[ios-app-settings-logging]
+image:ios-app-settings-logging.png[ios-app-settings-logging, width=40%]
 
 === Locating iPhone & iPad App Crash Logs
 
-In the worst case scenario, when the app either isn't responding or is crashing, iOS saves a crash log on the device.
-You can find it under menu:Settings[Privacy > Analytics > Analytics Data].
+In the worst case scenario, when the app either isn't responding or is crashing, iOS saves a crash log
+on the device. You can find it under menu:Settings[Privacy > Analytics > Analytics Data].
```

Comment:
Why there is a new line after log? Shouldn't this in one line?

Reply:
no worry, this renders continuous as one line.
only a special coding at the end of a line forces a linebreak or a blank line between two lines does so.